components/inspectit-ocelot-configurationserver/src/main/java/rocks/inspectit/ocelot/security/audit/AuditEventListener.java

```diff
@@ -0,0 +1,33 @@
+package rocks.inspectit.ocelot.security.audit;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.PostPersist;
+import javax.persistence.PostRemove;
+import javax.persistence.PostUpdate;
+
+/**
+ * Listener for events to be audit logged.
+ */
+@Slf4j
+public class AuditEventListener {
```

Comment:
I have still kept EventListener as it offers more granular events for JPA based entities like User. For example, it makes distinction among CREATE & UPDATE as against SAVE.
Please let me know if you want this removed.